dts/arm/st/stm32f2.dtsi

```diff
@@ -137,6 +137,60 @@
 				label = "GPIOI";
 			};
 		};
+
+		usart1: serial@40011000 {
+			compatible = "st,stm32-usart", "st,stm32-uart";
+			reg = <0x40011000 0x400>;
+			clocks = <&rcc STM32_CLOCK_BUS_APB2 0x00004000>;
+			interrupts = <37 0>;
+			status = "disabled";
+			label = "UART_1";
```

Comment:
USART_1

Reply:
In the already committed ‘*.dtsi’ files for stm32fx the usartn and uartn are labelled with “UART_n”.
If this is a new requirement, I think that the usart6 should be labelled with “USART_6” too.

Reply:
This is right, my bad. Let's keep UART for now. Will remove these comments